app/views/shared/_linear_approval_status.html.erb

```diff
@@ -0,0 +1,46 @@
+  <div class="col-md-12">
+    <table class="col-md-12 data_container" border='0'>
+      <tr class="email-status">
+        <td colspan="2" padding="10">
+          <table id="status-container-detail" class="col-md-12">
+            <tr class="status-header-container">
+              <td>
+                <strong>Approval Status</strong>
+              </td>
+              <td>
+                <%= status_icon_tag(@cart.status) %>
+                <strong><%= @cart.status %></strong>
+                (<%= @cart.number_approved %> of <%= @cart.total_approvers %> approved)
+              </td>
+            </tr>
+            <tr class="status-header-container-detail">
+              <td colspan="2">
+                <table class="approval_details data_container">
+                  <%- last_status = nil %>
+                  <%- @cart.approvals_in_list_order.each_with_index do |approval,index| %>
+                  <tr class="approval-row">
+                    <td>
+                      <%- if approval.status != last_status %>
+                      <strong><%= approval.status %></strong>
+                      <%- end %>
+                    </td>
+                    <td
+                    <%= generate_bookend_class(index, @cart.approvals_in_list_order.count) %>
+                    >
+                    <%= status_icon_tag(approval.status) %>
+                    <ul>
+                      <li><%= approval.user_full_name %></li>
+                      <li><%= mail_to approval.user_email_address %></li>
+                    </ul>
+                  </td>
+                </tr>
+                <%- last_status = approval.status %>
+                <%- end %>
+              </table>
+            </td>
+          </tr>
+        </table>
+      </td>
+    </tr>
+  </table>
+</div>
```

Comment:
The email_status template used to be shared but we split them up into web and email versions. The rationale was that we can refactor the web version to a table-free version that would more easily work with responsive markup. @sashax, additional thoughts?